### PR TITLE
feat: move docs to docs.safe.global

### PR DESCRIPTION
**Pull request by tanay1337**
Please don't merge until https://github.com/safe-global/safe-docs/pull/400/files is in production.

### CODE REVIEW

**Comment by tanay1337** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by tanay1337** [comment]
recheck

File: README.md [hunk 1]
```diff
@@ -7,36 +7,33 @@
...
-## Using with docker
+## Using Docker
 
-If you have **Docker** installed on your system, you can just run:
+**Prerequisite:** Install [Docker Desktop](https://www.docker.com/products/docker-desktop/).
 
-```bash
+Once Docker is installed on your system, run the following command to create Safe Smart Accounts:
+
+```
```
Comment (Uxio0) [inline README.md:22]: Why do you remove the bash code annotation?

File: README.md [hunk 2]
```diff
@@ -7,36 +7,33 @@
...
 
-# Safe-CLI
+# Safe CLI
 
-Command line utility for **Safe** contracts. Use it to manage your **Safe** easily from the command line.
+Safe CLI is a command-line utility for Safe contracts. You can use it to manage your Safe Smart Account from the command line.
 
-Safe-CLI does not rely on _Safe{Core} API_, so it can be used in networks where Safe services are not available. If they are available,
-Safe-CLI can also interact with them in `tx-service` mode.
+It does not rely on Safe{Core} API and can also be used in networks where Safe services are unavailable. Learn more through the [documentation](https://docs.safe.global/advanced/cli-overview).
```
Comment (Uxio0) [inline README.md:14]: Why do you remove the bolding and italic?

File: README.md [hunk 3]
```diff
@@ -7,36 +7,33 @@
...
+```
 docker run -it safeglobal/safe-cli safe-creator
 ```
 
-for creating Safes
-
-or
-
-```bash
+You can also run the following command to run the Safe CLI:
```
Comment (Uxio0) [inline README.md:26]:
```suggestion
You can also run the following command to run the Safe CLI with a existing Safe:
```

File: README.md [hunk 4]
```diff
@@ -7,36 +7,33 @@
...
+```
 docker run -it safeglobal/safe-cli safe-cli
```
Comment (Uxio0) [inline README.md:28]:
```suggestion
docker run -it safeglobal/safe-cli safe-cli <checksummed_safe_address> <ethereum_node_url>
```

File: README.md [hunk 5]
```diff
@@ -7,36 +7,33 @@
...
 ```
 
-to run the actual **safe-cli**
+## Using Python PIP
 
-## Installing
+**Prerequisite:** [Python](https://www.python.org/downloads/) >= 3 (Python 3.10 is recommended).
```
Comment (Uxio0) [inline README.md:33]: Python higher than 3.7 is mandatory, not 3

File: README.md (hunk 1 above)
Comment (tanay1337) [reply README.md:22]: Fixed

File: README.md (hunk 2 above)
Comment (tanay1337) [reply README.md:14]: This has been copied from the docs, keeping them in line with the doc's style guide.

**Review by Uxio0 — APPROVED** [review]

File: README.md [hunk 6]
```diff
@@ -7,36 +7,33 @@
...
+It does not rely on Safe{Core} API and can also be used in networks where Safe services are unavailable. Learn more through the [documentation](https://docs.safe.global/advanced/cli-overview).
 
-## Using with docker
+## Using Docker
 
-If you have **Docker** installed on your system, you can just run:
+**Prerequisite:** Install [Docker Desktop](https://www.docker.com/products/docker-desktop/).
 
-```bash
+Once Docker is installed on your system, run the following command to create Safe Smart Accounts:
```
Comment (germartinez) [inline README.md:20]:
```suggestion
Once Docker is installed on your system, run the following command to create new Safe accounts:
```

File: README.md [hunk 7]
```diff
@@ -7,36 +7,33 @@
 ![Python 3.12](https://img.shields.io/badge/Python-3.12-blue.svg)
 [![Docker Image Version (latest semver)](https://img.shields.io/docker/v/safeglobal/safe-cli?label=Docker&sort=semver)](https://hub.docker.com/r/safeglobal/safe-cli)
 
-# Safe-CLI
+# Safe CLI
 
-Command line utility for **Safe** contracts. Use it to manage your **Safe** easily from the command line.
+Safe CLI is a command-line utility for Safe contracts. You can use it to manage your Safe Smart Account from the command line.
```
Comment (germartinez) [inline README.md:12]: I see "The Safe Smart Account" as the protocol, and "Safe account" as a user account.

```suggestion
Safe CLI is a command-line utility for Safe contracts. You can use it to manage your Safe account from the command line.
```

**Review by germartinez — CHANGES_REQUESTED** [review]

File: README.md [hunk 8]
```diff
@@ -7,35 +7,32 @@
...
+docker run -it safeglobal/safe-cli safe-cli <checksummed_safe_address> <ethereum_node_url>
 ```
 
-to run the actual **safe-cli**
-
-## Installing
+## Using Python PIP
 
-**Python >= 3.7** is required. **Python 3.10** is recommended.
+**Prerequisite:** [Python](https://www.python.org/downloads/) >= 3.7 (Python 3.10 is recommended).
```
Comment (Uxio0) [inline README.md:33]:
```suggestion
**Prerequisite:** [Python](https://www.python.org/downloads/) >= 3.9 (Python 3.12 is recommended).
```

**Review by Uxio0 — APPROVED** [review]

File: README.md (hunk 8 above)
Comment (tanay1337) [reply README.md:33]: Thanks, updating this is the docs as well.

**Comment by tanay1337** [comment]
@Uxio0 Okay to be merged!

**Pull request merged**